addons/l10n_din5008_purchase/report/din5008_purchase_order_templates.xml

```diff
@@ -48,6 +48,63 @@
                 </tr>
             </t>
 
+            <t t-set="din5008_document_title">
+                <span t-if="o and o._name == 'purchase.order'">
+                    <t t-if="o.state in {'draft', 'sent', 'to approve'}">Request for Quotation</t>
+                    <t t-elif="o.state in {'purchase', 'done'}">Purchase Order</t>
+                    <t t-elif="o.state == 'cancel'">Cancelled Purchase Order</t>
+                </span>
+            </t>
+        </xpath>
+    </template>
+    <template id="report_purchasequotation_document" inherit_id="purchase.report_purchasequotation_document">
+        <xpath expr="//t[@t-set='address']" position="after">
+            <t t-set="din5008_document_information">
```

Comment:
If i'm not mistaken the t-set of this template are the same as the ones for the purchase order document, could we avoid duplicating the code by using some another template that we would call :thinking: but maybe we would lose the possibility to modify it from the technical menu could you check ? :smile: